src/tools/miri/tests/pass-dep/concurrency/tls_pthread_drop_order.rs

```diff
@@ -53,6 +53,7 @@ unsafe extern "C" fn dtor(ptr: *mut u64) {
     // The correct sequence is: First key 0, then key 1, then key 0.
     // Note that this relies on dtor order, which is not specified by POSIX, but seems to be
     // consistent between Miri and Linux currently (as of Aug 2022).
+    #[allow(leading_zeros_in_decimal_literals)]
```

Comment:
Please use `expect` so that we remember to remove the annotation if we ever refactor this code.